protocol/serialization-types/src/external_transition.rs

```diff
@@ -37,5 +37,5 @@ pub struct ExternalTransition {
 pub struct ExternalTransitionV1(pub Versioned<ExternalTransition, 1>);
 
 impl bin_prot::encodable::BinProtEncodable for ExternalTransitionV1 {
-    const PREALLOCATE_BUFFER_BYTES: usize = 13 * 1024;
+    const PREALLOCATE_BUFFER_BYTES: usize = 13 * 1024; // FIXME: This size is not suitable for blocks with snark work data
```

Comment:
I suggest removing this trait entirely since it is only a helper for writing tests. Feel free to increase the buffer to make things work (but it might break again if we get a larger block)

Reply:
increased the buffer size for now, to support all the sample test blocks we have